app/lib/dags/host_filter.json.erb

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "host_filtering",
```

Comment:
We should use the same names that we defined in constants as dag stage names, e.g. DAG_NAME_HOST_FILTER. Can we passed them to the template?

Reply:
Sure — since they're created in https://github.com/chanzuckerberg/idseq-web/pull/2427, I'll wait for that to merge in first

Reply:
Added in constants!